Project.toml

```diff
@@ -101,15 +102,15 @@ ComponentArrays = "b0b7db55-cfe3-40fc-9ded-d10e2dbeff66"
 DynamicQuantities = "06fc5a27-2a28-4c7c-a15d-362465fb6821"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 Groebner = "0b43b601-686d-58a3-8a1c-6623616c7cd4"
+LambertW = "984bce1d-4616-540c-a9ee-88d1112d94c9"
```

Comment:
why is this needed?

Reply:
https://github.com/JuliaSymbolics/Symbolics.jl/blob/6271c8bd56f2266176eb93dfd4fcd2c9f50bedec/test/solver.jl#L524

It's used in a test but never added as a test dep. I'm guessing it used to be transitively loaded, but isn't now.